api/PerformanceObserver.json

```diff
@@ -225,6 +225,51 @@
           }
         }
       },
+      "supportedEntryTypes": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PerformanceObserver/supportedEntryTypes",
+          "support": {
+            "chrome": {
+              "version_added": "73"
+            },
+            "chrome_android": {
+              "version_added": "73"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
```

Comment:
Seems to be supported in opera, webkit nightly (thought not safari yet)  and ff.  didn't test edge.

Reply:
I've added Opera, but set them to true because my confidence in my knowledge of Opera is declining.